torch/quantization/convert_modules.py

```diff
@@ -125,17 +174,12 @@ def convert_to_quantized(module):
 
     return module_swapped
 
-DEFAULT_SWAP_MODULE_MAPPING = {
-    torch.nn.Linear: nnq.Linear,
-    torch.nn.ReLU: nnq.ReLU,
-}
-
 STUB_MODULE_MAPPING = {
     QuantStub: nnq.Quantize,
     DeQuantStub: nnq.DeQuantize
 }
 
-def swap_module(mod, mapping=DEFAULT_SWAP_MODULE_MAPPING):
+def swap_module(mod, mapping):
```

Comment:
Can we combine STUB_MODULE_MAPPING and DEFAULT_MODULE_MAPPING into one?

Reply:
after changing the order I think we can do that, since all modules will have observer